spec.html

```diff
@@ -24753,10 +24785,10 @@ <h1>Error Handling and Language Extensions</h1>
   <p>An implementation shall report all errors as specified, except for the following:</p>
   <ul>
     <li>
-      Except as restricted in <emu-xref href="#sec-forbidden-extensions"></emu-xref>, an implementation may extend |Script| syntax, |Module| syntax, and regular expression pattern or flag syntax. To permit this, all operations (such as calling `eval`, using a regular expression literal, or using the Function or RegExp constructor) that are allowed to throw *SyntaxError* are permitted to exhibit implementation-defined behaviour instead of throwing *SyntaxError* when they encounter an implementation-defined extension to the script syntax or regular expression pattern or flag syntax.
+      Except as restricted in <emu-xref href="#sec-forbidden-extensions"></emu-xref>, an implementation may extend |Script| syntax, |Module| syntax, and regular expression pattern or flag syntax. To permit this, all operations (such as calling `eval`, using a regular expression literal, or using the Function or RegExp constructor) that are allowed to throw *SyntaxError* are permitted to exhibit host-defined behaviour instead of throwing *SyntaxError* when they encounter a host-defined extension to the script syntax or regular expression pattern or flag syntax.
     </li>
     <li>
-      Except as restricted in <emu-xref href="#sec-forbidden-extensions"></emu-xref>, an implementation may provide additional types, values, objects, properties, and functions beyond those described in this specification. This may cause constructs (such as looking up a variable in the global scope) to have implementation-defined behaviour instead of throwing an error (such as *ReferenceError*).
+      Except as restricted in <emu-xref href="#sec-forbidden-extensions"></emu-xref>, an implementation may provide additional types, values, objects, properties, and functions beyond those described in this specification. This may cause constructs (such as looking up a variable in the global scope) to have host-defined behaviour instead of throwing an error (such as *ReferenceError*).
```

Comment:
It's a little odd here that the _implementation_ is providing additional types but this leads to _host_-defined behavior. Perhaps "a host or implementation may provide [...]"?

Reply:
Done.